opm/simulators/flow/BlackoilModelEbos.hpp

```diff
@@ -558,8 +558,6 @@ namespace Opm {
                         Scalar tmpSat = saturationsNew[phaseIdx] - saturationsOld[phaseIdx];
                         resultDelta += tmpSat*tmpSat;
                         resultDenom += saturationsNew[phaseIdx]*saturationsNew[phaseIdx];
-                        assert(std::isfinite(resultDelta));
-                        assert(std::isfinite(resultDenom));
```

Comment:
I these are not finite then we have a problem somewhere and I'd rather be told about it.  The mechanism probably should not be `assert()` though.  On another note, PR #4856 changes this area too so we might want to coordinate this with that PR.

Reply:
For now, let's leave these two in, and we'll consider it with #4856 instead.